examples/tests/valid/captures.w

```diff
@@ -16,6 +16,14 @@ let handler = inflight (event: str): str => {
   for stuff in bucket1.list() {
     print(stuff);
   }
+
+  print(bucket2.public_url("file.txt"));
+
+  try {
+    bucket1.public_url("file.txt");
```

Comment:
Excellent! Please also add one invalid test (where the bucket is not public and `.public_url()` throws an exception).

Reply:
it seems like I can't... it catches only errors that are emitted from the compiler itself... I'll add an issue instead...

Reply:
https://github.com/winglang/wing/issues/1808